beaglebone/servo/src/__main__.py

```diff
@@ -0,0 +1,79 @@
+import time
+import Adafruit_BBIO.PWM as PWM
+import lcm
+from rover_msgs import ServoCmd
+
+
+# Don't know if this is the same for the servo we're using
+SERVO_MAX_DC = 10.0
+SERVO_MIN_DC = 4.0
+
+amino_blue = "P9_14"
+ammonia_blue = "P9_16"
+
+amino_white = "P8_13"
+ammonia_white = "P8_19"
+
+amino_yellow = "P9_21"
+ammonia_yellow = "P9_22"
+
+lcm_ = lcm.LCM()
+
+
+def angle_to_dc(degrees):
+    percent = degrees / 120.0
+    dc = SERVO_MIN_DC + (percent * (SERVO_MAX_DC - SERVO_MIN_DC))
+    return dc
+
+
+def run_servo(pin, degrees):
+    dc = angle_to_dc(degrees)
+    PWM.set_duty_cycle(pin, dc)
+
+
+def servo_init(pin, degrees):
+    dc = angle_to_dc(degrees)
+    PWM.start(pin, dc, 50)
+
+
+def servocmd_callback(channel, msg):
+    servo = ServoCmd.decode(msg)
+    # Blue callback
+    if (servo.id == "amino_blue"):
+        run_servo(amino_blue, servo.position)
+        time.sleep(4)

```

Comment:
what's with these sleeps?

Reply:
idk jonathan put them in, i can get rid of them i guess?